src/Apps/Feature/Components/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from "./FeatureHeader"
+export * from "./FeatureFeaturedLink"
+export * from "./FeatureSet"
```

Comment:
Use of an index file _could_ make sense here, but is it necessary for something that will only contain three files? I don't have a strong opinion -- just thinking editor ergonomics and how to reduce the number of matched `index` files in a search or tab bar set.